tests/FakeItEasy.Tests/Core/SequentialCallContextTests.cs

```diff
@@ -0,0 +1,146 @@
+﻿namespace FakeItEasy.Tests.Core
+{
+    using System.Collections.Generic;
+    using FakeItEasy.Core;
+    using FakeItEasy.Tests.TestHelpers;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class SequentialCallContextTests

```

Comment:
> More generally, is it offering anything over and above `OrderedCallMatchingSpecs`?

If a change is to be made, I'm all for this class disappearing in favour of `OrderedCallMatchingSpecs`